onnx/test/symbolic_shape_test.py

```diff
@@ -16,9 +16,9 @@
 
 class TestSymbolicShape(unittest.TestCase):
 
-    def _assert_valueInfo_shape(self, onnx_model, valueInfo):
+    def _assert_valueinfo_shape(self, onnx_model, valueinfo):  # type: (ModelProto, List[ValueInfoProto]) -> None
```

Comment:
Doesn't seem to handle the case where inference did not infer a type for a name specified in ```valueinfo``` correctly. I think it may be better to treat this method as a shorthand for invoking ```get_shape_from_name``` and equality check for every entry in ```valueinfo```.

Reply:
Good catch. I have considered the case if the name does not exist and reuse get_shape_from_name here.